package.json

```diff
@@ -51,7 +51,6 @@
     "@types/node": "^12.6.6",
     "@typescript-eslint/eslint-plugin": "^1.13.0",
     "@typescript-eslint/parser": "^1.13.0",
-    "babel-eslint": "^10.0.2",
```

Comment:
could you send a separate PR rewriting `no-large-snapshots`'s tests to not use `babel-eslint` to reduce the diff here? I like the simplification, but this PR is pretty huge 🙂

Reply:
I've broken this out into #367